modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/render.tsx

```diff
@@ -86,6 +86,12 @@ export default function render(
 
 		container.classList.add('lfr-tooltip-scope');
 
+		const children = container.querySelectorAll('.tag-body-content > *');
+
```

Comment:
we may save this dom query if we manage to find a way to tell `reactRenderer.renderReact()` what are the children. This way we would not even need a marker class. A possibility to ewxplore is to directrly prepare the `children` props in the backend. Would this work?